Feature/ks8500 runner 1.14.0

thedao01 opened the pull request:
modified public.yaml and README.md according to recommendations from JESUS MACIAS PORTELA <jesus.maciasportela@telefonica.com>

thedao01 committed:
Update public.yaml

thedao01 committed:
Update README.md

thedao01 committed:
Update README.md

thedao01 committed:
Update README.md

thedao01 committed:
Update README.md

thedao01 committed:
Update Dockerfile

thedao01 committed:
Update changelog.md

thedao01 committed:
Update changelog.md

alvarocurt commented:
Hi The,
Thank you so much for the fast update, but unfortunatelly the PR doesn't fit the format we pursue.
Please add the following changes:
- It is not necessary to add documentation changes in the changelog. You ought to mention only the upgrade in the software version.
- The dependency should be to tn_bastion better than tn_init. Apologizes because this was discussed after the template was shared, but all components that don't depend to another one are better off with tn_bastion as their base.
- Adding the component name as the tag is not useful. I added some useful ones for all components, so you should not worry about them.
- README is nice. This is totally up to the developer (not used by the LLM) so no comments there.

thedao01 committed:
Update changelog.md

thedao01 committed:
Update changelog.md

thedao01 committed:
Update public.yaml

thedao01 committed:
Update README.md

thedao01 commented:
Hi Alvaro

I have fixed the changes according to your comments.
Please recheck it to see if the fixes are correct now.
Br.
The

From: Álvaro Curto Merino ***@***.***>
Sent: Wednesday, May 14, 2025 10:36 AM
To: 6G-SANDBOX/6G-Library ***@***.***>
Cc: The Dao ***@***.***>; Author ***@***.***>
Subject: Re: [6G-SANDBOX/6G-Library] Feature/ks8500 runner 1.14.0 (PR #135)

alvarocurt left a comment (6G-SANDBOX/6G-Library#135) Hi The, Thank you so much for the fast update, but unfortunatelly the PR doesn't fit the format we pursue. Please add the following changes: It is not necessary to add documentation changes
ZjQcmQRYFpfptBannerStart
This Message is From an External Sender: Use caution opening files, clicking links or responding to requests.
ZjQcmQRYFpfptBannerEnd
[Image removed by sender.]alvarocurt left a comment (6G-SANDBOX/6G-Library#135)<https://urldefense.com/v3/__https:/github.com/6G-SANDBOX/6G-Library/pull/135*issuecomment-2879311955__;Iw!!I5pVk4LIGAfnvw!ki3fn3ucuGmcRtotroItCSg7YPGUmSoRvPCLE8QRxgk7H_I6UwG40d61EuR-UTCZiJj1EO9y416EvmtffHk6XbWB$>

Hi The,
Thank you so much for the fast update, but unfortunatelly the PR doesn't fit the format we pursue.
Please add the following changes:

  *   It is not necessary to add documentation changes in the changelog. You ought to mention only the upgrade in the software version.
  *   The dependency should be to tn_bastion better than tn_init. Apologizes because this was discussed after the template was shared, but all components that don't depend to another one are better off with tn_bastion as their base.
  *   Adding the component name as the tag is not useful. I added some useful ones for all components, so you should not worry about them.
  *   README is nice. This is totally up to the developer (not used by the LLM) so no comments there.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/6G-SANDBOX/6G-Library/pull/135*issuecomment-2879311955__;Iw!!I5pVk4LIGAfnvw!ki3fn3ucuGmcRtotroItCSg7YPGUmSoRvPCLE8QRxgk7H_I6UwG40d61EuR-UTCZiJj1EO9y416EvmtffHk6XbWB$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/A74P7QBE5TPZXJOWEU2FFBD26L57NAVCNFSM6AAAAAB4TB5EU2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQNZZGMYTCOJVGU__;!!I5pVk4LIGAfnvw!ki3fn3ucuGmcRtotroItCSg7YPGUmSoRvPCLE8QRxgk7H_I6UwG40d61EuR-UTCZiJj1EO9y416EvmtffMHSehWm$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
